.php-cs-fixer.dist.php

```diff
@@ -85,11 +85,10 @@
     ],
     'yoda_style' => false,
     'php_unit_test_case_static_method_calls' => false,
+    'phpdoc_summary' => false,
```

Comment:
Don't think we should activate it

Reply:
I will open a follow up PR to showcase after this Pr

Reply:
Cool, thanks!